WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  Event: <br /><%= select_tag(:event, options_for_select(@events, selected: params[:event], disabled: "")) %>
+</div>
+
+<div class="form-group">
+  Region: <br /><%= select_tag(:region, content_tag(:option, "All", value: "all") + grouped_options_for_select(@regions, params[:region])) %>
+</div>
+
+<div class="form-group">
+  Years: <br /><%= select_tag(:years, options_for_select(@years, selected: params[:years], disabled: "")) %>
+</div>
+
+<div class="form-group">
+  Name, City or Venue: <br /><%= text_field_tag "search" %>
+</div>
+
+<div class="form-group">
+  <%= button_tag(type: 'submit', name: "display", value: "List", class: "btn btn-info") do %>
+    <i class="glyphicon glyphicon-list"></i> List
+  <% end %>
+</div>
+
+<div class="form-group">
```

Comment:
Would it make sense to put both of these buttons in the same `.form-group`? I don't love how this looks on mobile:

![2016-02-04_939x1218_kaladin](https://cloud.githubusercontent.com/assets/277474/12831969/148a7506-cb4c-11e5-9a3e-9b603e4c3f9f.png)

Have you tested this on a mobile phone? I"m nervous that the floating header will end up taking up all the available space, always.